voice-interface/src/main/java/com/derpgroup/derpwizard/voice/model/SsmlDocumentBuilder.java

```diff
@@ -249,6 +250,15 @@ public String toString() {
     return buildString();
   }
 
+  public boolean isConversationEnd() {
```

Comment:
To tell the implementing resource whether it should treat this as the final response in the conversation.